upload.go

```diff
@@ -134,6 +134,20 @@ func Configure() (*TarUploader, *Prefix, error) {
 		upload.StorageClass = storageClass
 	}
 
+	serverSideEncryption, ok := os.LookupEnv("WALG_S3_SSE")
+	if ok {
+		upload.ServerSideEncryption = serverSideEncryption
+	}
+
+	sseKmsKeyId, ok := os.LookupEnv("WALG_S3_SSE_KMS_ID")
+	if ok {
+		upload.SSEKMSKeyId = sseKmsKeyId
+	}
+
+	if serverSideEncryption == "aws:kms" && sseKmsKeyId == "" {
```

Comment:
I think we should add some validation code here.
If user have sseKmsKeyId but no serverSideEncryption WAL-G should print an error, if serverSideEncryption is unknown, WAL-G should print warning.